docs/src/pages/demos/snackbars/ConsecutiveSnackbars.js

```diff
@@ -1,104 +1,117 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { withStyles } from '@material-ui/core/styles';
+import React, { useState } from 'react';
```

Comment:
As far as I know, there is no bundle size advantage. So, better improve the DX. It's easier to import once.

```suggestion
import React from 'react';
```